tests/smoke/gmres.cpp

```diff
@@ -693,6 +831,6 @@ int main( int argc, char **argv ) {
 	std::cout << std::endl;
 
 	// done
-	return out.rc;
+	return (50 + out.rc);
```

Comment:
Unless out.rc is zero, then this should return 0 also, not 50. So in fact this code should read something like

`return (out.rc == 0 ? 0 : (50 + out.rc));`